types/googlemaps/index.d.ts

```diff
@@ -286,9 +287,11 @@ declare namespace google.maps {
     }
 
     enum MapTypeControlStyle {
-        DEFAULT,
-        DROPDOWN_MENU,
-        HORIZONTAL_BAR,
+        DEFAULT = 0,
+        DROPDOWN_MENU = 1,
+        HORIZONTAL_BAR = 2,
```

Comment:
Wrong `DROPDOWN_MENU` & `HORIZONTAL_BAR` value.

![image](https://user-images.githubusercontent.com/10235949/67053044-e006eb80-f148-11e9-82eb-5e0560689602.png)

Reply:
Thanks, fixed those and sorted them according to the enums. I was trying to keep the previous ordering (by name) in place, but yeah it obviously confused me a bit 🤓